ramda.d.ts

```diff
@@ -1551,8 +1551,8 @@ declare module R {
          *      var isUpperCase = function(val, key) { return key.toUpperCase() === key; }
          *      R.pickWith(isUpperCase, {a: 1, b: 2, A: 3, B: 4}); //=> {A: 3, B: 4}
          */
-        pickWith<T>(pred: (v: any, k: string) => boolean, T): T;
-        pickWith<T>(pred: (v: any, k: string) => boolean): (T) => T;
+        pickWith<T>(pred: (v: any, k: string) => boolean, T: any): T;
+        pickWith<T>(pred: (v: any, k: string) => boolean): (T: any) => T;
 
```

Comment:
T is the type of an argument that is not there. So it should be ...boolean, obj: T): ...